bin/eslint.js

```diff
@@ -7,7 +7,12 @@ var exitCode = 0,
 
 if (useStdIn) {
     process.stdin.pipe(concat({ encoding: "string" }, function(text) {
-        exitCode = cli.execute(process.argv, text);
+        try {
+            exitCode = cli.execute(process.argv, text);
+        } catch (ex) {
+            console.error(ex.message);
+            exitCode = 1;
+        }
```

Comment:
No, the error is thrown in the callback, not in the process of reading the stream.